torch/distributed/_composable/fsdp/_fsdp_common.py

```diff
@@ -150,3 +151,16 @@ def _cast_fp_tensor(dtype: torch.dtype, x: torch.Tensor) -> torch.Tensor:
     ):
         return x
     return x.to(dtype)
+
+
+def _get_device_handle_from_device(device: Optional[torch.device] = None):
```

Comment:
let us not use a default arg if we never expect to call this without passing a `device`